### DISCUSSION TITLE
Arch linux or Arch based distros supported ?

### DISCUSSION THREAD

**Discussion by Greatz08** [ripple/rippled#5208] [General]
> Operating System: macOS, Windows (64-bit), or most Linux distributions (Red Hat, Ubuntu, and Debian supported).

In docs these requirements are mentioned and since no arch linux is mentioned so thought to ask about it as it is also quite popular and i use it too :-) .
Btw what was the need to mention Ubuntu and Debian when Ubuntu is already debian based ?

**Comment by ximinez** [comment]
Where did you find the quoted text? I don't see it occurring anywhere in this repository.

**Reply by Greatz08** [reply]
![image](https://github.com/user-attachments/assets/e3151549-11c5-46f9-a0ac-05c04e2811e0)
https://xrpl.org/docs/infrastructure/installation/system-requirements
official website docs

**Reply by legleux** [reply]
Arch Linux most definitely works as I and a few others use it but it is not officially supported.

I have a PKGBUILD for the AUR in the works for the `develop` branch that still needs a few issues ironed out.

Let me know if you come across any issues/problems.
